pkgs/development/python-modules/pytest-examples/default.nix

```diff
@@ -59,6 +59,10 @@ buildPythonPackage rec {
     # Test fails with latest ruff v0.1.2
     # See https://github.com/pydantic/pytest-examples/issues/26
     "test_ruff_error"
+    # Tests fails with python3.12 as they use deprecated functions
+    # Fixed upstream but not packed as a release yet
```

Comment:
Can we fetch the commits with the fixes?

Reply:
Updated the package to the current HEAD of its upstream master branch and now the tests succeed again.

Reply:
I assume that using https://github.com/pydantic/pytest-examples/commit/551ba911713c2859caabc91b664723dd6bc800c5 as patch would be enough.

Updating to unstable would also need an adjustment of the version as it would no longer be 0.0.10.

Reply:
Without patch [pydantic/pytest-examples@3bef5d6](https://github.com/pydantic/pytest-examples/commit/3bef5d644fe3fdb076270833768e4c6df9148530) the tests tests/test_run_examples.py::test_run_example_ok_fail and tests/test_run_examples.py::test_run_directly still fail. I now changed the rev back and added the two patches.